Update search-cloud-based-mailboxes-for-on-premises-users.md (#3959)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/search-cloud-based-mailboxes-for-on-premises-users.md

- The user whose primary mailbox is located in the on-premises organization must be assigned a Microsoft Teams license and a minimum of an Exchange Online Plan 1 license.

> - Have an exchange hybrid configuration in order there is no duplicate mailbox generated for the user which his mailbox is hosted on-premises or sync the exchange schema.

Reviewer: ```suggestion
- The user must have an Exchange hybrid configuration in order to avoid duplicate mailboxes from being generated if the mailbox is hosted on-premises or synchronized with the Exchange schema.
```
